src/instructlab/model/download.py

```diff
@@ -66,7 +66,7 @@ def download(self):
         Download specified model from Hugging Face
         """
         click.echo(
-            f"Downloading model from Hugging Face: {self.repository}@{self.release} to {self.download_dest}..."
+            f"Downloading model from Hugging Face:\n    Model: {self.repository}@{self.release}\n    Destination: {self.download_dest}"
```

Comment:
Suggestion: Create a constant for the indention spaces:

DEFAULT_INDENT = "    "